crates/ursa-network/src/service.rs

```diff
@@ -609,6 +612,11 @@ where
                             }
                         }
                         RequestType::StoreSummary(cache_summary) => {
+                            if self.peer_cached_content.len() == self.max_cache_summaries {
+                                let key_to_remove =
+                                    *self.peer_cached_content.iter().next().unwrap().0;
```

Comment:
```suggestion
                                    *self.peer_cached_content.keys().next().unwrap();
```

Reply:
https://github.com/fleek-network/ursa/pull/429#discussion_r1129889565

Reply:
wow @kckeiks beat me to it